docs/pages/usage/proving.md

```diff
@@ -0,0 +1,142 @@
+# Running Stone Prover
+
+This guide provides step-by-step instructions on how to use the Stone Prover to create and verify proofs for Cairo programs. Stone Prover is a tool developed by StarkWare to facilitate proof generation and verification in the Cairo programming environment. This guide is based on the official [Stone Prover documentation](https://github.com/starkware-libs/stone-prover?tab=readme-ov-file#creating-and-verifying-a-proof-of-a-cairo-program).
+
+## Creating and Verifying a Proof
+
+To create and verify a proof using Stone Prover, follow these steps:
+
+#### 1. Clone the Stone Prover repository:
+
+```bash
+git clone https://github.com/starkware-libs/stone-prover.git
+cd stone-prover
+```
+
+#### 2. Build the prover
+
+Use Bazel to build the Stone Prover executable:
+
+```bash
+bazel build //src/main/prover:stone_prover
```

Comment:
No need to build, you can use static binaries https://github.com/dipdup-io/stone-packaging?tab=readme-ov-file#creating-and-verifying-a-test-proof-using-binaries